src/app/tests.py

```diff
@@ -1106,40 +1108,94 @@ def test_error_archive_license_requests(self):
     @skipIf(not getAccessToken() and not getGithubUserId() and not getGithubUserName(), "You need to set gihub parameters in the secret.py file for this test to be executed properly.")
     def test_archive_license_requests_feature(self):
         """Check if the license is shifted to archive requests when archive button is pressed"""
+        TEST_LOGIN_INFO = {
+        "provider": "github",
+        "uid": str(getGithubUserId()),
+        "access_token": getAccessToken(),
+        "login": getGithubUserName(),
+        "id": getGithubUserId(),
+        "password": 'pass'
+        }
+        # login first
+        self.user = User.objects.create(username=TEST_LOGIN_INFO["login"],
+                                        is_active=True,
+                                        is_superuser=True)
+        self.user.set_password(TEST_LOGIN_INFO["password"])
+        self.user.save()
+        social_auth = UserSocialAuth.objects.create(provider=TEST_LOGIN_INFO["provider"],
+        uid=TEST_LOGIN_INFO["uid"],
+        extra_data=TEST_LOGIN_INFO,
+        user=self.user)
+        self.user = authenticate(username=TEST_LOGIN_INFO["login"],
+                                 password=TEST_LOGIN_INFO["password"])
+        login = self.client.login(username=TEST_LOGIN_INFO["login"],
+                                  password=TEST_LOGIN_INFO["password"])
```

Comment:
Let's create a TestUtil for common code to setup github login.

Reply:
A separate class or a function inside the ArchiveLicenseRequestsViewsTestCase?

Reply:
@rtgdk Should I include this in `utils.py` or a separate file named `testUtils.py`

Reply:
Let's have `testUtils.py` or keep the util inside `tests.py` itself.

Reply:
@rtgdk done